### PR TITLE
Add use case for Find Command in Developer Guide

### PR DESCRIPTION
**Pull request by jamietan2002**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -523,6 +523,51 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+1. User starts FoodWhere.
+2. User enters the command to find food stalls.
+3. FoodWhere displays the food stalls that matches the search keywords.
+
+   Use case ends.
+
+**Extensions**
+
+* 2a. User uses at least one of the prefixes as a search term:
+    * `n/` for Name search.
```
Comment (clarence-chew) [inline docs/DeveloperGuide.md:539]: I think exact prefixes are too specific for use case.

**Review by clarence-chew — COMMENTED** [review]
I have a concern.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (jamietan2002) [reply docs/DeveloperGuide.md]: Fixed.

**Review by MinHeinA — APPROVED** [review]
LGTM!

**Pull request merged**